pkg/webircgateway/client_command_handlers.go

```diff
@@ -315,7 +315,13 @@ func (c *Client) ProcessLineFromClient(line string) (string, error) {
 			}
 
 			message.Params[1] = strings.Join(newCaps, " ")
-			line = message.ToLine()
+			if strings.Trim(message.Params[1], " ") == "" {
+				// We fake this because otherwise some servers never respond.
+				c.SendClientSignal("data", "CAP * ACK :message-tags")
```

Comment:
`message-tags` should be `c.RequestedMessageTagsCap` here as the tag could include some type of draft or versioned cap name.